homeworks/week02/README.md

```diff
@@ -0,0 +1,232 @@
+# Week 2, Homework
+
+For this homework you will write code that takes some data and renders
+it into a web page, using functions, objects and arrays. The data
+will be a list of movies with some information about each. You will
+receive this in the form of an array of simple JavaScript objects,
+which your code will convert into HTML. You will also add some simple
+CSS to style the page.
+
+To begin, clone this repo to your local machine. You should be able to
+open the ```index.html``` file in your browser and see a simple
+```Hello, world.``` message.
+
+Open and examine the ```index.html``` file. You will see a function with the following signature:
+
+```javascript
+function render(movieArray, displayWide)
+```
+
+This function will be automagically called for you when you open or
```

Comment:
let's leave it as is. kind of a cheesy term, fairly easy to intuit, would make a bigger deal than its worth to explain I think